roles/contiv/templates/netplugin.j2

```diff
@@ -0,0 +1,6 @@
+{% if contiv_encap_mode == "vlan" %}
+NETPLUGIN_ARGS='-vlan-if {{ netplugin_interface }} -ctrl-ip {{ netplugin_ctrl_ip }} -plugin-mode kubernetes -cluster-store etcd://{{ etcd_url }}'
+{% endif %}
+{% if contiv_encap_mode == "vxlan" %}
```

Comment:
@rajatchopra 

1) Added a README file under roles/contiv.  Anyway still need proper documentation in the Openshift docs repo which we will add in the coming days/ weeks.

2) Briefly mentioned in this README but referred to Openshift docs for full details

3) Squashed

4) We retain kube-proxy for some Openshift scenarios

5) Not straightforward to disable VXLAN mode specially since it can be used in some limited scenarios specially with native kubernetes over cloud. Documentation clearly states that only VLAN mode and bare metal deployment is supported currently for Openshift + Contiv.  Will remove these restrictions in coming weeks.